Source/scrollrt.cpp

```diff
@@ -1979,7 +1979,7 @@ void DrawZoom(int x, int y)
 	}
 
 	/// ASSERT: assert(gpBuffer);
-	gpBufEnd = &gpBuffer[PitchTbl[143]];
+	gpBufEnd = &gpBuffer[PitchTbl[-17 + SCREEN_Y]];

```

Comment:
do you know what the logic is suppose to be here?

Reply:
Yes. When drawing normally, the range is 0-352. When drawing zoomed, everything except the lowest tile is rendered on the upper screen. So the coordinates (i.e. range) is (-17, 160). Where it draws 17 pixels above the normal screen.

Reply:
> Yes. When drawing normally, the range is 0-352. When drawing zoomed, everything except the lowest tile is rendered on the upper screen. So the coordinates (i.e. range) is (-17, 160). Where it draws 17 pixels above the normal screen.

So many treasure troves of information in the GitHub issue comments.